Include <asm/ptrace.h> to get necessary definitions on AArch64

In glibc commit
https://sourceware.org/git/?p=glibc.git;a=commit;h=7d05a8168b45c0580e1f9a79c2dd26c8f0d31fca,
including <asm/ptrace.h> from <sys/user.h> on AArch64 has been
removed. So the Google Breakpad code, which used to build fine on
glibc 2.18 (CodeSourcery toolchain for example), no longer builds with
glibc 2.19 (Linaro toolchain for example).

To fix this, this patch adds the missing <asm/ptrace.h> include (for
AArch64 only, to be conservative).

This patch has not been submitted upstream, since more recent versions
of Google Breakpad have completely changed this part of the code.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: b/src/client/linux/minidump_writer/linux_dumper.h
===================================================================
--- a/src/client/linux/minidump_writer/linux_dumper.h
+++ b/src/client/linux/minidump_writer/linux_dumper.h
@@ -43,6 +43,9 @@
 #include <stdint.h>
 #include <sys/types.h>
 #include <sys/user.h>
+#if defined(__aarch64__)
+#include <asm/ptrace.h>
+#endif
 
 #include "common/memory.h"
 #include "google_breakpad/common/minidump_format.h"
